guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -299,11 +296,11 @@ inlined into the script.
 ### Running Tests
 
 It is not customary in Rails to run the full test suite before pushing
-changes. The railties test suite in particular takes a long time, and takes an
+changes. The railties test suite, in particular, takes a long time and takes an
 especially long time if the source code is mounted in `/vagrant` as happens in
 the recommended workflow with the [rails-dev-box](https://github.com/rails/rails-dev-box).
 
-As a compromise, test what your code obviously affects, and if the change is
+As a compromise, test what your code directly affects, and if the change is
```

Comment:
I think "obviously" is the right word here because we're asking the reader to do something based on what is obvious to them.

```suggestion
As a compromise, test what your code obviously affects, and if the change is
```